packages/dialtone-vue2/recipes/conversation_view/editor/editor.vue

```diff
@@ -32,15 +32,16 @@
               :active="$refs.richTextEditor?.editor?.isActive(button.selector)"
               size="xs"
               :aria-label="button.tooltipMessage"
+              :class="{ 'd-btn--icon-only': !button.label }"
```

Comment:
You shouldn't have to set this manually... something is wrong in the button component I think. Looking into it.

Reply:
Pushed a fix for this, you should be able to remove this now.

Reply:
Fixed for real this time.